pkg/api/mlflow/api/request/expiriment.go

```diff
@@ -0,0 +1,57 @@
+package request
+
+// ExperimentTagPartialRequest is a partial request object for different requests.
+type ExperimentTagPartialRequest struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// ExperimentPartialRequest is a partial request object for different requests.
+type ExperimentPartialRequest struct {
+	ID               string                        `json:"experiment_id"`
+	Name             string                        `json:"name"`
+	ArtifactLocation string                        `json:"artifact_location"`
+	LifecycleStage   string                        `json:"lifecycle_stage"`
+	LastUpdateTime   int64                         `json:"last_update_time"`
+	CreationTime     int64                         `json:"creation_time"`
+	Tags             []ExperimentTagPartialRequest `json:"tags"`
+}
+
+// CreateExperimentRequest is a request object for `POST mlflow/create` endpoint.
```

Comment:
```suggestion
// CreateExperimentRequest is a request object for `POST mlflow/experiments/create` endpoint.
```

Same remarks for all the comments below ⬇️

Reply:
done.